packages/icons/svgo-jsx.config.ts

```diff
@@ -33,8 +37,8 @@ export const config: Config = {
   template,
   transformFilename,
   svgProps: {
-    width: "16",
```

Comment:
Problem, see for example https://github.com/webstudio-is/webstudio-designer/pull/943/files#diff-feb10459fc59f9de0ebfcab6e8dd0ba63b0c3a7390489ffa967d55dcab1dea0f

play.svg

It is designed at 22px size, when we run the compiler, it sets mixed sizes on it, it ends up partially 16 and partially 22, I think we set width/height but not viewport or something

Reply:
I am not sure I can just set 16 on that icon manually, it will probably break icons that are designed at 22

Reply:
> It is designed at 22px size

So do we want to have a separate set of icons for topbar and the rest of the app?

Reply:
> I think we set width/height but not viewport or something

and it's fine. If we change viewport we'll need to scale all paths in icon

Reply:
> I am not sure I can just set 16 on that icon manually, it will probably break icons that are designed at 22

they will be scaled

Reply:
> So do we want to have a separate set of icons for topbar and the rest of the app?

Separate set yes, toolbar only or not not sure, I guess its not just toolbar

Reply:
taylor informed me yesterday that this is basically a standard to have separate sets for different purposes

Reply:
Let's generate separate set then